src/scenes/login/Login.js

```diff
@@ -18,29 +19,81 @@ const styles = StyleSheet.create({
     fontSize: 24,
     marginBottom: 20,
   },
+  form: {
+    fontSize: 24,
+    marginBottom: 20,
+    backgroundColor: 'lime',
+  },
+
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  loginContainer: {
+    width: '80%',
+    alignItems: 'center',
+    padding: 10,
+    elevation: 10,
+    backgroundColor: '#e6e6e6',
+  },
+  textInput: {
+    height: 40,
+    width: '100%',
+    margin: 10,
+    backgroundColor: 'white',
+    borderColor: 'gray',
+    borderWidth: StyleSheet.hairlineWidth,
+    borderRadius: 10,
+  },
 })
 
-const Profile = ({ navigation }) => (
+const Login = () => (
   <View style={styles.root}>
     <StatusBar barStyle="light-content" />
-    <Text style={styles.title}>Profile</Text>
-    <Button
-      title="Go to Details"
-      color="white"
-      backgroundColor={colors.lightPurple}
-      onPress={() => {
-        navigation.navigate('Details', { from: 'Profile' })
-      }}
-    />
+    <Text style={styles.title}>Ekran logowania do aplikacji</Text>
+    <View style={styles.loginContainer}>
+      <Text>Login Screen</Text>
+      <Formik
+        initialValues={{ email: '', password: '' }}
+        onSubmit={(values) => console.log(values)}
+      >
+        {({
+          handleChange, handleBlur, handleSubmit, values,
+        }) => (
+          <>
+            <TextInput
+              name="email"
+              placeholder="Email Address"
+              style={styles.textInput}
+              onChangeText={handleChange('email')}
+              onBlur={handleBlur('email')}
+              value={values.email}
+              keyboardType="email-address"
+            />
+            <TextInput
+              name="password"
+              placeholder="Password"
+              style={styles.textInput}
+              onChangeText={handleChange('password')}
+              onBlur={handleBlur('password')}
+              value={values.password}
+              secureTextEntry
+            />
+            <Button onPress={handleSubmit} title="Submit" />
+          </>
+        )}
+      </Formik>
```

Comment:
RC: Create forms folder in src path and add LoginForm component. It will be easiest to mantain in the feature. Also please create LoginForm.styles.js and move all styles related to form styles